src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -949,3 +950,21 @@ def UpdatePermission(func):  # noqa: N802
     """
     _set_func_permissions(func, Permission.UPDATE)
     return func
+
+
+# Empty request body decorator
+
+def allow_empty_body(func):  # noqa: N802
+    """
+    The POST/PUT request methods decorated with ``@allow_empty_body``
+    are allowed to send empty request body.
+    """
+    try:
+        func._cp_config['tools.json_in.force'] = False
+    except (AttributeError, KeyError):
+        func._cp_config = {'tools.json_in.force': False}
+
+    if not isinstance(func._cp_config, dict):
```

Comment:
Remove the following 3 lines because the same thing is already done in the try-catch block above.
```suggestion
```

Reply:
Oh..missed that!